cstrml/market/src/lib.rs

```diff
@@ -719,12 +718,19 @@ impl<T: Config> Module<T> {
         UsedTrashSizeII::mutate(|value| {*value = 0;});
     }
 
+
     fn upsert_new_file_info(cid: &MerkleRoot, extend_replica: bool, amount: &BalanceOf<T>, curr_bn: &BlockNumber, file_size: u64) {
         // Extend expired_on or expected_replica_count
         if let Some((mut file_info, used_info)) = Self::files(cid) {
             let prev_first_class_count = file_info.reported_replica_count.min(file_info.expected_replica_count);
+            // expired_on < claimed_at => not live. Only for new file
```

Comment:
1. Not begin yet
2. Already closed(wait to be put into trash or refreshed)
3. On going(Normal)